### DISCUSSION TITLE
AOI on PV2

### DISCUSSION THREAD

**Discussion by georgebratu** [FirstGearGames/FishNet#524] [Feature Request]
Hi,

I want AOI (area of interest) on PV2 (client prediction ver 2). For example on game Dota2, there is FOW (fog of war). If an enemy hero is not visibile on the map, then me, as a player, I shouldn't receive updates (inputs and reconciliation) from server regarding that enemy state until it gets revealed on the map. The revealing process itself needs to start with a reconciliation. Reason are because I want to prevent players from cheating (they shouldn't receive information about where the other players are hiding) and also save bandwidth.

**Comment by FirstGearGames** [comment]
There are no plans to add AOI specific to prediction. Our observer system already determines what objects can be seen/receive data, there's not much benefit to adding the complexity this requests would require.
